pkg/controllers/user/logging/generator/project_template.go

```diff
@@ -204,12 +204,10 @@ var ProjectTemplate = `{{range $i, $store := .projectTargets -}}
       {{ if $val.SharedKey}}
       shared_key {{$val.SharedKey}}
       {{end -}}
-      {{ if $val.Username}}
-      username  {{$val.Username}}
-      {{end -}}
-      {{ if $val.Password}}
-      password  {{$val.Password}}
-      {{end -}}
+      {{- if and $val.Username $val.Password}}
+      username {{$val.Username}}
+      password {{$val.Password}}
+      {{- end }}
       weight  {{$val.Weight}}
```

Comment:
Please change line 62-66 for elasticsearch, not here(here is for fluentd forward)